posthog/templates/message_preferences/preferences.html

```diff
@@ -0,0 +1,103 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="max-w-lg mx-auto py-8 px-4">
```

Comment:
Don't love this but for now its fine. Long term we should have this in the posthog app tbh or some proper frontend as the things we will want to do will become more and more complex